sdcm/cluster_baremetal.py

```diff
@@ -108,7 +108,7 @@ def node_setup(self, node, verbose=False):
         We have to modify scylla.yaml on our own because we are not on AWS,
         where there are auto config scripts in place.
         """
-        self._node_setup(node, verbose)
+        pass  # self._node_setup(node, verbose)
```

Comment:
Is this intended?
Won't it affect our support of bare metal?

Reply:
it's intended, bare metal doesn't have this function... looks like it's actually fixes baremetal support...
We'll need to revisit baremetal support, I don't think it was used in a while...